game/src/challenges/cutscene.rs

```diff
@@ -89,36 +89,33 @@ struct CutscenePlayer {
 
 impl State<App> for CutscenePlayer {
     fn event(&mut self, ctx: &mut EventCtx, app: &mut App) -> Transition {
-        match self.panel.event(ctx) {
-            Outcome::Clicked(x) => match x.as_ref() {
-                "quit" => {
-                    // TODO Should SandboxMode use on_destroy for this?
-                    app.primary.clear_sim();
-                    app.set_prebaked(None);
-                    return Transition::Multi(vec![Transition::Pop, Transition::Pop]);
-                }
-                "back" => {
-                    self.idx -= 1;
-                    self.panel =
-                        make_panel(ctx, &self.name, &self.scenes, &self.make_task, self.idx);
-                }
-                "next" => {
-                    self.idx += 1;
-                    self.panel =
-                        make_panel(ctx, &self.name, &self.scenes, &self.make_task, self.idx);
-                }
-                "Skip cutscene" => {
-                    self.idx = self.scenes.len();
-                    self.panel =
-                        make_panel(ctx, &self.name, &self.scenes, &self.make_task, self.idx);
-                }
-                "Start" => {
-                    return Transition::Pop;
-                }
-                _ => unreachable!(),
-            },
-            _ => {}
-        }
+        if let Outcome::Clicked(x) = self.panel.event(ctx) { match x.as_ref() {
```

Comment:
This is definitely a place where `cargo +nightly fmt` can cleanup

Reply:
I forgot to reformat this. But sure, `cargo fmt` is easier, I did know you were using it frequently.

Reply:
I see you used stable; we're actually using `cargo +nightly fmt` just for formatting, because it has some settings: https://github.com/a-b-street/abstreet/blob/master/rustfmt.toml

Originally the motivation was to fix things like https://github.com/a-b-street/abstreet/commit/95a84fd26ac4cbf3fb439fe0de8ff01453dc0a40#diff-8dd8cf08580aa074427224cf8e370b132f1fdf0626cdc5701f9218957d7fbfec. But I just tried removing these nightly-only overrides, using long lines in `tutorial.rs` again, and formatting everything with `cargo fmt`, and it looks fine with stable. So this is a great opportunity to ditch the dependency on the nightly toolchain just for formatting.

So if you don't mind in this PR, just remove `rustfmt.toml` entirely. I'll update the separate docs repo after this PR is merged, and we can all start using `cargo fmt`.

Reply:
Sure, I'll remove the config.

I mostly used stable because I assumed that is what you were doing. (Also, using nightly cargo fmt might be problematic if developers use different nightlies.)